setup.py

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python
 import os
 import subprocess
-from distutils.errors import CCompilerError, DistutilsExecError, DistutilsPlatformError
+from distutils.errors import CompileError
```

Comment:
removing this requires either a very new setuptools version or using the wildcard `BaseException`.